app/helpers/application_helper.rb

```diff
@@ -25,4 +25,20 @@ def add_or_remove_from_queue_button(user, video)
       link_to 'X Remove From Queue', queue_item_path(user.video_queue_item(video)), method: :delete, class: 'btn btn-default', id: 'remove-button'
     end
   end
+
+  def display_last_4(card)
+    "****-****-****-#{card.last4}"
```

Comment:
Remember not every card carries the same numeric groupings. I think mastercard starts with a group of 5 for example.